src/main/java/com/iexec/worker/tee/scone/LasServicesManager.java

```diff
@@ -69,11 +70,14 @@ public boolean startLasService(String chainTaskId) {
     }
 
     /**
-     * "iexec-las-0xWalletAddress-timestamp" as lasContainerName to avoid naming conflict
+     * Create LAS container name with a random salt to avoid naming conflicts
      * when running multiple workers on the same machine or using multiple SMS.
+     * Note: Starting from 64 characters, `failed to lookup address
+     * information: Name does not resolve` error occures.
      */
     String createLasContainerName() {
-        return "iexec-las-" + workerConfigService.getWorkerWalletAddress() + "-" + new Date().getTime();
+        return "iexec-las-" + workerConfigService.getWorkerWalletAddress() + "-" 
```

Comment:
Does the `substring` helps to stay under the 64 characters length ?

Reply:
In this context yes :)